frontend/src/libraries/explorer-wamp/details.ts

```diff
@@ -0,0 +1,73 @@
+import { ExplorerApi } from ".";
+
+export interface Details {
+  accountsCount: number;
+  totalNodesCount: number;
+  onlineNodesCount: number;
+  lastDayTxCount: number;
+  lastBlockHeight: number;
+  transactionsPer10Second?: number;
+  transactionsPerSecond?: number;
+}
+
+export default class DetailsApi extends ExplorerApi {
+  async getDetails(): Promise<Details> {
+    try {
+      const blockTimestamp = await this.call("select", [
+        `SELECT timestamp FROM blocks WHERE timestamp > ((strftime('%s','now') - 30) * 1000 ) ORDER BY timestamp DESC LIMIT 1`
+      ]).then((it: any) => (it[0].timestamp !== null ? it[0].timestamp : null));
```

Comment:
Are you sure it is going to be `null`? I would expect not even a single element in the `it` array, and thus `undefined` value  accessing the 0th element.